HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -84,6 +84,49 @@ def customizeHLTfor47017(process):
 
     return process
 
+
+def customizeHLTfor47079(process):
+    """Remove unneeded parameters from the HLT menu"""
+    for filt in filters_by_type(process, 'PrimaryVertexObjectFilter'):
+        if hasattr(filt, 'filterParams') and hasattr(filt.filterParams, 'pvSrc'):
+            del filt.filterParams.pvSrc  # Remove the pvSrc parameter
+
+    for prod in producers_by_type(process, 'HcalHitReconstructor'):
+        # Remove "digiTimeFromDB" if "Subdetector" is not "HF"
+        if hasattr(prod, 'Subdetector') and getattr(prod, 'Subdetector') != "HF":
+            if hasattr(prod, 'digiTimeFromDB'):
+                delattr(prod, 'digiTimeFromDB')
```

Comment:
Same here ??

Reply:
what seems to be the problem? Am I supposed to guess it ?

Reply:
Same as my comment on your most recent PR #47107 (ie, removal of some top-level parameter depending on value/existance of other top-level parameters). A ConfDb  template has an exact set of top-level parameters, so if instances need to be made with different top-level parameters (HF or nor HF), that will not work.

Reply:
> Same as my comment on your most recent PR 

there is no such comment.

Reply:
Addressed at https://github.com/mmusich/cmssw/commit/e214fa7c30b34549c91d1c55fcaacef4d803c4a2.